app/src/main/java/edu/uc/hickmadc/customnotifications/dto/Alarm.kt

```diff
@@ -1,7 +1,13 @@
 package edu.uc.hickmadc.customnotifications.dto
 
-data class Alarm (var name : String, var desc : String, var active : Boolean, var random : Boolean, var alarm_id:Int=0) {
```

Comment:
I had a hard time understanding the rest of the code until I took a look at these two classes... Having variable names that are the same for two classes can mix things up, so I just renamed them accordingly.

Technical suggestion: try to use better naming conventions all around, I think everyone would benefit from this.

Reply:
I disagree on this one.  The attributes are all inside the class "Alarm", which gives it context.  If you repeat the name "alarm" in each variable name, that's redundant... which is extra typing, reading.